src/FNA3D_Driver_D3D11.c

```diff
@@ -4017,6 +4021,16 @@ static void D3D11_AddDisposeEffect(
 		renderer->currentPass = 0;
 		renderer->effectApplied = 1;
 	}
+
+	// invalidate all inputLayouts
+	for (int i = 0; i < renderer->inputLayoutCache.count; i += 1)
```

Comment:
`int i` needs to be declared at the top of the function

Reply:
updated